jupyter/trustyai/ubi9-python-3.11/Pipfile

```diff
@@ -3,11 +3,21 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 name = "pypi"
 
+[[source]]
+url = "https://download.pytorch.org/whl/cu121"
+verify_ssl = true
+name = "pytorch"
+
 [dev-packages]
 
 [packages]
+# PyTorch packages
+torch = {version = "~=2.2.1", index = "pytorch"}
```

Comment:
Note: we should check how much the new images are bigger and whether we're okay with that.

Reply:
@jstourac According to a cursory look this increase the image size (on the registry) from ~1GiB to ~5Gib.